api/src/main/java/marquez/db/JobDao.java

```diff
@@ -278,6 +296,34 @@ JobRow upsertJob(
       UUID symlinkTargetId,
       PGobject inputs);
 
+  default JobRow upsertJob(
+      UUID uuid,
+      JobType type,
+      Instant now,
+      UUID namespaceUuid,
+      String namespaceName,
+      String name,
+      String description,
+      UUID jobContextUuid,
+      String location,
+      UUID symlinkTargetId,
+      PGobject inputs) {
+    UUID jobUuid =
+        upsertJobNoParent(
```

Comment:
Wouldn't we want the `upsertJobNoParent()` call to the `JobRow` object similar to other upsert calls?

Reply:
Sorry, do you mean that the `upsertJobNoParent` query would go back to `RETURNING *` instead of `RETURNING uuid`? If that's what you mean, I made this change so that the subsequent `findJobByUuidAsRow` call queries the `jobs_view` - returning the FQN rather than the simple name.